section2/reflection.md

```diff
@@ -1,15 +1,36 @@
 ## Section 2 Reflection
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
+- I noticed that both resources said mistakes are inevitable. For me this is easy to think about, but more difficult to deal with when the mistakes are actually happening. Another thing I appreciated from the video is the idea that the people she admires aren't just extremely smart and gifted people, but that they have worked really hard and experience has paid off. I think it is unfair to just assume anyone is naturally good at anything, because it really just minimizes all the work they put in. A third thing that resonated with me was the idea of making mental action notes. This is a great idea, and something that might really be helpful for me is not just a mental action plan, but actually writing it down.
+
+---
 
 2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
+- One thing I have really been improving on in the last few years is creating awareness of how I do/think about things. I used to get into a negative thought process and spiral ever deeper, but now I am much better at catching it, and not spiraling. One thing I have a difficult time with is seeing the mistakes as an opportunity for growth. I think with Turing I'm just so worried about getting behind, that I get so anxious about time. I need to remember the great resources Turing offers and make sure to utilize them!
```

Comment:
I can spiral, too, and get so caught up in the fact that I'm "behind" that I then actually fall behind. As you go into Mod 1, I would keep in mind that most students always feel some level of behind - and that's an ok place to be at Turing. Don't worry about if your peers are ahead. It is very uncomfortable, but do what you can to get yourself into a place where you accept that and let you brainspace go into moving forward, one tiny step at a time.